server/models/video/video-channel.ts

```diff
@@ -311,6 +311,17 @@ export type SummaryOptions = {
               ')'
             ),
             'viewsPerDay'
+          ],
+          [
+            literal(
+              '(' +
+              'SELECT COALESCE(SUM("video".views), 0) AS totalViews ' +
+              'FROM "video" ' +
+              'WHERE "video"."channelId" = "VideoChannelModel"."id" ' +
+              'GROUP BY "video"."channelId" ' +
```

Comment:
Why do you `GROUP BY`?

Reply:
That... looks useless? I guess?
I'll remove it, see what it does.